### DISCUSSION TITLE
[Question] Criteria for adding bindings

### DISCUSSION THREAD

**Discussion by pentamassiv** [gtk-rs/gtk-rs-core#1175] [General]
Is there a place where I can find out which criteria are used to add bindings? I did not find it on the website or this repo. Is it whenever anyone asks about it or do you try to provide as many as possible or something else?

I am asking because I thought about looking for more bindings that exist in the sys crate but not in the safe wrapper and opening PRs.

**Comment by sdroege** [comment]
Anything that makes sense to add can be added. If there are equivalent (or better) APIs for the same functionality in `std`, for example, then we wouldn't add bindings here though. Similarly, if there are better Rust implementations for the same feature and the bindings are not needed for interoperability with other API.
